lib/purchases_flutter.dart

```diff
@@ -985,6 +986,21 @@ class Purchases {
     );
   }
 
+  /// Displays the specified store in-app message types to the user if there are any available to be shown.
+  /// - Important: This should only be used if you disabled these messages from showing automatically
+  /// during SDK configuration setting ``shouldShowInAppMessagesAutomatically`` to ``false``.
+  ///
+  /// @param [types] The types of messages to show.
+  static Future<void> showInAppMessages({
```

Comment:
Should we add this to the API tests for flutter? Both with and without the parameter.

Reply:
Already added some to `test/purchases_flutter_test.dart` (last file in the list). Anything extra I should add?

Reply:
I think we also need to add these to the API tests. For example: https://github.com/RevenueCat/purchases-flutter/blob/main/api_tester/lib/api_tests/purchases_flutter_api_test.dart#L7